snap/snapcraft.yaml

```diff
@@ -1,36 +1,56 @@
-name: saber
-version: 0.9.4
-summary: Handwritten notes
-description: A cross-platform libre handwritten notes app, available for Android, iOS, Linux, Windows, and macOS!
+name: saber # you probably want to 'snapcraft register <name>'
+base: core18 # the base snap is the execution environment for this snap
+version: '0.9.4' # just for humans, typically '1.2+git' or '1.3.2'
```

Comment:
Please make these changes (i.e. wrapping with single-quotes) to `./apply_version.sh` to pass the tests

![image](https://user-images.githubusercontent.com/21128619/220866388-ab52dd27-945f-4426-a10e-b57596fdeec1.png)

Reply:
Can you send me a suggestion to commit about this? I'm not getting this file in my repo.

Reply:
This is the problem I'm pointing. As there will be issue of tags and release if I use the fork repo of mine. Or the thing I have to do is, have a separate repo only for the snap. Would that be good option?